tactic/ring.lean

```diff
@@ -7,6 +7,7 @@ Evaluate expressions in the language of (semi-)rings.
 Based on http://www.cs.ru.nl/~freek/courses/tt-2014/read/10.1.1.61.3041.pdf .
 -/
 import algebra.group_power tactic.norm_num
+import tactic.converter.interactive
```

Comment:
Would it make more sense to import `tactic.ring`, `tactic.norm_num`, etc. from `tactic.converter.interactive`, and add all of the `conv` tactics in that file?

Reply:
Yes, I thought it would be more expected behaviour if to use `norm_num` in a `conv` block the user would have to `import tactic.norm_num`, just as they would outside the block.

Also I didn't want `import tactic.converter.interactive`, giving access to some basic `conv` addons, to silently include a bunch of potentially "big" tactics which happen to be made available inside `conv`.

Reply:
I see. I never use `conv`, so it somehow feels less fundamental than `norm_num` to me. It feels natural to keep `conv` tactics all located in one place, and  there's not much of a cost to importing `norm_num` and `ring` by default. But this is barely worth worrying about.

Note, if `tactic.norm_num` imports `tactic.converter.interactive`, you only need to import the former here. (This is also barely worth worrying about.)

Reply:
For me the distinction is that`ring` and `norm_num` (particularly) are about numbers and hence only relevant in certain specialized corners of mathematics. :-) Whereas `conv` is just part of the essential toolkit of working with dependent type theory, and therefore essential everywhere.